core/global/keyboard_instance.gd

```diff
@@ -19,6 +19,13 @@ var context: KeyboardContext
 # keyboard inputs should go, and how to handle submits.
 func open(ctx: KeyboardContext) -> void:
 	set_context(ctx)
+	if context and context.target:
+		# Show the caret
+		if context.target is LineEdit:
+			var line_edit := context.target as LineEdit
+			line_edit.caret_force_displayed = true
+		# Don't allow the node to grab focus
```

Comment:
Why? Please explain the context instead of just saying that the code says.

Reply:
Added context